src/app/containers/App/index.test.jsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
-import ReactRouter from 'react-router-dom';
+import * as ReactDOMRouter from 'react-router-dom';
 import { ClientApp, ServerApp } from '.';
 import * as App from './App';
 
 jest.mock('./App', () => jest.fn(() => <>Mocked App component</>));
 
+jest.mock('react-router-dom', () => {
```

Comment:
I think we should only mock for tests that need mocks

Reply:
You could revert all changes in this file and do this:

```js
jest.mock('react-router-dom', () => ({
  ...jest.requireActual('react-router-dom'),
}));
```

Reply:
In two tests before, we were mocking `StaticRouter` and `BrowserRouter`, so we could make sure they got called with the right arguments.

The problem is that in `react-router-dom` v5.1.1, `StaticRouter` is now exported with only a get accessor, so we can no longer mock it, the way we were:

```jsx
ReactRouter.StaticRouter = jest.fn(() => <></>);
```

Instead, using `jest.mock('react-router-dom', ...)`, we can mock the entire module, rather than trying to set `StaticRouter`.

However, we can no longer use jest.fn(() => <></>) as the mocked value of `StaticRouter`, because jest.mock can only be used “globally” to all tests, and other tests depend on the original implementation of `StaticRouter` and `BrowserRouter`.

When I inspected those tests, I saw that they only make sure that the rendered element contains some text.

So, using `jest.fn(({ children})  => <>{children}</a>)` should render the children props, letting the text get rendered, making it a worth replacement mock for both `StaticRouter` and `BrowserRouter`.

Reply:
An alternative is to separate those two tests, into their own file, where we can `jest.mock(...)` as we please, so the other tests can use the original implementations of `StaticRouter` and `BrowserRouter`.

Reply:
Following your suggestion on slack, this has been fixed.